SPEC.md

```
@@ -916,25 +919,29 @@ Compatibility profile:
 
 Subprocess launch parameters:
 
-- Base command: `claude.command` (default: `claude`)
+- Base command: `claude.command` (default: `claude`), interpreted as a shell command string.
 - First turn invocation:
   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)